apps/settings/elements/messaging.html

```diff
@@ -23,8 +23,14 @@ <h2 data-l10n-id="message-confirmation"></h2>
       <ul class="mmsSettings-list">
         <li id="menuItem-readReport" aria-disabled="true">
           <gaia-switch name="ril.mms.requestReadReport.enabled"/>
-            <label data-l10n-id="message-read-reports"></label>
-            <details data-l10n-id="message-read-reports-details"></details>
+            <label data-l10n-id="message-request-read-reports"></label>
+            <details data-l10n-id="message-request-read-reports-details"></details>
+          </gaia-switch>
+        </li>
+        <li id="menuItem-sendReadReport" aria-disabled="true">
+          <gaia-switch name="message.mms.sendReadReport.enabled"/>
```

Comment:
nit: I'd use "messages" instead of "message" as prefix; "messages" is the official name for our application